packages/react-vapor/src/components/editor/tests/CodeEditor.spec.tsx

```diff
@@ -81,30 +80,34 @@ describe('CodeEditor', () => {
         });
 
         it('should call onChange prop when its value prop changes', () => {
-            const onChangeSpy: jasmine.Spy = jasmine.createSpy('onChange');
+            const onChangeSpy = jasmine.createSpy('onChange');
             const expectedValue: string = 'the expected value';
 
             mountWithProps({onChange: onChangeSpy});
-            codeEditor.setProps({value: expectedValue});
+
+            codeEditor
+                .find(ReactCodeMirror.Controlled)
+                .first()
+                .props()
+                .onChange({} as CodeMirror.Editor, undefined, expectedValue);
 
             expect(onChangeSpy).toHaveBeenCalledTimes(1);
             expect(onChangeSpy).toHaveBeenCalledWith(expectedValue);
         });
 
-        it(`should clear codemirror's history if we set a new value`, () => {

```

Comment:
Why did you delete this test?

Reply:
Honestly, not for a good reason. Because this component is now connected to Redux, it broke every tests and I couldn't reached the private properties as it was done before. Finally I figured out a way to pass this test.